tools/preflight/suite_test.go

```diff
@@ -0,0 +1,76 @@
+package preflight
+
+import (
+	"context"
+	"fmt"
+	"os"
+	goruntime "runtime"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientGoScheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest/printer"
+)
+
+var (
+	ctx           = context.Background()
+	k8sClient     client.Client
+	testEnv       = &envtest.Environment{}
+	envTestScheme *runtime.Scheme
+	logger        *logrus.Logger
+	run           *Run
+)
+
+func TestPreflight(t *testing.T) {
+	RegisterFailHandler(Fail)
+	junitReporter := reporters.NewJUnitReporter("junit-preflight-unit-test.xml")
+	RunSpecsWithDefaultAndCustomReporters(t,
+		"webhook-server",
+		[]Reporter{printer.NewlineReporter{}, junitReporter})
+}
+
+var _ = BeforeSuite(func() {
+	goos := goruntime.GOOS
+	goarch := goruntime.GOARCH
+	kubeBuilderPath := fmt.Sprintf("/tmp/kubebuilder_2.3.2_%s_%s/bin", goos, goarch)
```

Comment:
Move binaries from `/tmp` to `/etc/` and use default exported path for these binaries so that it'll not search in `/tmp` for local dev setups

Reply:
made the changes in script accordingly. removed this from here. It will use default binaries from `/usr/local/kubebuilder/bin/` path